impartus.go

```diff
@@ -488,11 +488,11 @@ func CreateTempM3U8File(downloadedPlaylist DownloadedPlaylist) M3U8File {
 		defer secondView.Close()
 
 		secondView.WriteString(`#EXTM3U
-#EXT-X-VERSION:3
-#EXT-X-MEDIA-SEQUENCE:0
-#EXT-X-ALLOW-CACHE:YES
-#EXT-X-TARGETDURATION:11
-#EXT-X-KEY:METHOD=NONE`)
+								#EXT-X-VERSION:3
+								#EXT-X-MEDIA-SEQUENCE:0
+								#EXT-X-ALLOW-CACHE:YES
+								#EXT-X-TARGETDURATION:11
+								#EXT-X-KEY:METHOD=NONE`)
```

Comment:
Undo this change. As this is a raw string, the output file will also contain the tabs that are part of this string.

Output file after this change:
![image](https://github.com/user-attachments/assets/369200a9-03ed-40dd-bb6b-345ab866b57d)

Previous file output:
![image](https://github.com/user-attachments/assets/e8876c11-f1ae-4996-a243-ac2d78368bbe)

This technically does not cause any bugs, but I prefer the old output.

Reply:
yeah sure, will do that.